### PR TITLE
Enhancement: RSpec tests

### PR DESCRIPTION
**Pull request by shrmnk**
## What
RSpec tests for controllers, models and routings

## Why
To ensure future development of the app does not break existing functionality

---

**Before submitting, check that:**

 - [X] You have added (passing) tests for your code.
 - [X] You have written good* commit messages.
 - [X] You have squashed relevant commits together.
 - [X] You have ensured that RuboCop is passing.
 - [X] Your PR relates to one subject, with a clear title and description using complete, grammatically correct sentences.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -91,6 +91,8 @@ group :test do
 
   # One-liner RSpec helper
   gem 'shoulda-matchers'
+
+  gem 'sidekiq'
```
Comment (tbssc) [inline Gemfile]: why did you add sidekiq?

File: spec/controllers/landing_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe LandingController, type: :controller do
+
+  describe 'GET index' do
+
+    it 'returns http success' do
+      get :index
```
Comment (tbssc) [inline spec/controllers/landing_controller_spec.rb]: move this to a `before` block. Then add a test that it renders index

Comment (shrmnk) [inline spec/controllers/landing_controller_spec.rb]: Addressed! https://github.com/tinkerbox/may-interns-twitter-clone-sherman/blob/475911c7ed1389cbca51e313ea5d68466b257ced/spec/controllers/landing_controller_spec.rb

File: Gemfile (hunk 1 above)
Comment (shrmnk) [inline Gemfile]: RSpec would not run otherwise
